sdk/batch/Microsoft.Azure.Batch.FileStaging/src/Microsoft.Azure.Batch.FileStaging.csproj

```diff
@@ -17,19 +17,25 @@
     </PackageReleaseNotes>
   </PropertyGroup>
   <PropertyGroup>
-    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
```

Comment:
Why are you changing this?

Reply:
We have a customer request to migrate the FileStaging and Conventions Batch SDKs to utilize the current Azure.Storage SDK. The Azure.Storage SDK is a Track 2 dependency library and our SDKs are currently track 1. In order to leverage the new current SDK, we had to drop support of some of the target track 1 frameworks (i.e. netstandard 1.4, net452) and explicitly specify net461 and netstandard2.0

Reply:
Is this not a breaking change?

Reply:
It is a breaking change. In order to support the Azure.Storage package reference, we've had to drop support for the netstandard1.4 and .net4.52 frameworks. These are deprecated, non-supported frameworks.

Reply:
/cc: @tg-msft, @KrzysztofCwalina do we need to follow any breaking change process if we drop support for a framework in track1 SDK?

Reply:
Given .NET has already abandoned those as well, I think it's probably fine to call it out as a breaking change in the release notes and point customers at a migration guide.

Reply:
Thank you @tg-msft. Given that information is added, I will go ahead and approve the PR.